app/assets/javascripts/analytics/error-tracking.js

```diff
@@ -0,0 +1,24 @@
+// Extension to track errors using google analytics as a data store.
+(function() {
+
+    "use strict";
+    var trackJavaScriptError = function (e) {
+      var errorSource = e.filename + ': ' + e.lineno;
+      _gaq.push([
+        '_trackEvent',
+        'JavaScript Error',
```

Comment:
Might be more consistent with other common events we have if this was 'javascript_error'.

That said, we've not got much consistency in event names: most use _ instead of space, though some use - and a few do use spaces.  Some are all lower-case, others are TitleCase.

For comparison some of our existing events are:

 - 'MS_guide'
 - 'IG_detailed_guidance'
 - 'Print Intent'
 - 'ScrollTo'
 - 'user_satisfaction_survey'
 - 'startpages'
 - 'multivariatetest_cohort_tax-disc-beta'

This is definitely a non-blocking comment, though. It would be good to set out some guidelines for event names in future (perhaps during the transition to universal analytics).